### DISCUSSION TITLE
Thing I made that relates to this.

### DISCUSSION THREAD

**Discussion by Steve0Greatness** [theforumhelpers/theforumhelpers.github.io#307] [Show and tell]
I've been working on a project for about a week now, it's something that lets you request to change your bio. A working version can be found [here](https://bio-change.stevesgreatness.repl.co/), and the [code is on Github too](https://github.com/Steve0Greatness/Forum-Helpers-Bio-Change-Request).

**Comment by Accio1** [comment]
Thanks for working on this. However, the next Forum Helpers Dashboard update, which I hope to release within the next week, will address this issue because it has a built in bio-changing feature.

**Reply by Steve0Greatness** [reply]
I know, it was just a fun project
